public/app/plugins/datasource/jaeger/_importedDependencies/utils/timeSrv.tsx

```diff
@@ -0,0 +1,42 @@
+import { dateMath, dateTime, DateTime, getDefaultTimeRange, isDateTime, TimeRange, TimeZone } from '@grafana/data';
+
+interface TimeModel {
+  fiscalYearStartMonth?: number;
+  getTimezone(): TimeZone;
+}
+
+export class TimeSrv {
```

Comment:
You may be able to do this a different way. Please see the `How to externalize a datasource` design doc, Gabor added a section to decouple the timesrv.